src/energyplus/ForwardTranslator/ForwardTranslateEnergyManagementSystemActuator.cpp

```diff
@@ -65,113 +69,203 @@ namespace openstudio {
+      const std::string spaceLoadNameForEMS = zoneOrSpaceName + " " + actuatedComponent_->nameString();
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentUniqueName, spaceLoadNameForEMS);
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentType, modelObject.actuatedComponentType());
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentControlType, modelObject.actuatedComponentControlType());
+      return idfObject;
+    }
 
-          return idfObject;
-        } else {
-          LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load.get().name().get()
-                                  << "' which is not associated with a ThermalZone, it will not be translated.");
-          return boost::none;
+    if (const boost::optional<ThermalZone> specificZone_ = modelObject.thermalZone()) {
+      auto spaces = specificZone_->spaces();
+      if (spaces.empty()) {
+        LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                << "' and specified that it wanted to be attached to the ThermalZone '" << specificZone_->nameString()
+                                << "' but the Thermal Zone has no Spaces, so it will not be translated.");
+        return boost::none;
+      }
+
+      std::string zoneOrSpaceName = specificZone_->nameString();
+
+      if (!m_forwardTranslatorOptions.excludeSpaceTranslation()) {
+        if (spaces.size() == 1) {
+          zoneOrSpaceName = spaces.front().nameString();
+          LOG(Warn, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                 << "' and specified that it wanted to be attached to the Thermal Zone '" << specificZone_->nameString()
+                                 << "' but you have turned on ForwardTranslation's Space Feature. Falling back to using the attached Space '"
+                                 << zoneOrSpaceName << "' since there is only 1.");
+        } else {  // if (spaces.size() > 1) {
+          std::sort(spaces.begin(), spaces.end(), WorkspaceObjectNameLess());
+          zoneOrSpaceName = spaces.front().nameString();
+          // TODO: do we grab the first space or just don't translate it?
+          LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                  << "' and specified that it wanted to be attached to the Thermal Zone '" << specificZone_->nameString()
+                                  << "' but you have turned on ForwardTranslation's Space Feature. Your Thermal Zone has multiple spaces "
+                                     "attached. Falling back to using the first Space '"
+                                  << zoneOrSpaceName << "'.");
         }
+      }
```

Comment:
We have a Zone, but we are translating to Spaces.

* If that Zone has one space, we gracefully recover by getting it and issue a simple warning.
* If that Zone has multiple spaces, we sort them by name, grab the first one, and issue an Error. @brianlball I have considered Throwing instead, thoughts?

Reply:
i think erroring is the right thing and giving as much info as possible.  a warn could just get ignored.